web-ui/components/EditorPanelComponents/Editor.vue

```diff
@@ -18,7 +18,7 @@ export default {
       const modifierKey = navigator.platform.match("Mac")
         ? e.metaKey
         : e.ctrlKey;
-      if (e.keyCode == 83 && modifierKey) {
+      if ((e.keyCode === 83 || e.key === "s") && modifierKey) {
```

Comment:
This should really be
```js
if ((e.key ? e.key === "s" : e.keyCode === 83) && modifierKey) {
```
As `keyCode` will likely always be set so there is not much point in checking it first.